Please note, the tests are failing on another module, seemingly unrelated:

```
2025-02-04 23:23:55,273 274 ERROR odoo odoo.addons.account_purchase_stock_report_non_billed.tests.test_account_purchase_stock_report_non_billed: FAIL: TestAccountPurchaseStockReportNonBilled.test_04_report_move_full_invoice_refund
Traceback (most recent call last):
  File "/__w/account-financial-reporting/account-financial-reporting/account_purchase_stock_report_non_billed/tests/test_account_purchase_stock_report_non_billed.py", line 119, in test_04_report_move_full_invoice_refund
    self.assertIn(move.id, domain_ids)
AssertionError: 87 not found in []
```

--- a/partner_statement/report/report_statement_common.py
+++ b/partner_statement/report/report_statement_common.py
@@ -353,7 +353,7 @@ class ReportStatementCommon(models.AbstractModel):
         aging_type = data["aging_type"]
         is_activity = data.get("is_activity")
         is_detailed = data.get("is_detailed")
-        today = fields.Date.today()
+        today = fields.Date.context_today(self)
         amount_field = data.get("amount_field", "amount")
 
         # There should be relatively few of these, so to speed performance

